src/Http/Http.Features/src/HeaderNames.cs

```diff
@@ -283,10 +283,16 @@ public static class HeaderNames
         /// <summary>Gets the <c>X-Frame-Options</c> HTTP header name.</summary>
         public static readonly string XFrameOptions = "X-Frame-Options";
 
+        /// <summary>Gets the <c>X-Powered-By</c> HTTP header name.</summary>
+        public static readonly string XPoweredBy = "X-Powered-By";
+
         /// <summary>Gets the <c>X-Requested-With</c> HTTP header name.</summary>
         public static readonly string XRequestedWith = "X-Requested-With";
 
+        /// <summary>Gets the <c>X-UA-Compatible</c> HTTP header name.</summary>
+        public static readonly string XUACompatible = "X-UA-Compatible";
+
         /// <summary>Gets the <c>X-XSS-Protection</c> HTTP header name.</summary>
-        public static readonly string XXssProtection = "X-XSS-Protection";
+        public static readonly string XXSSProtection = "X-XSS-Protection";
```

Comment:
This is a breaking change, do we need to deprecate the old version for a release cycle?

Reply:
Hasn't even gone out in preview, so isn't breaking?

Reply:
To clarify this was only added as a header 8 days ago

Reply:
Cool, I thought the original was already public! Thanks.